vcr_test.go

```diff
@@ -44,14 +34,7 @@ func vcrClientWrapper(mode, name string) (*rockset.RockClient, func(string) stri
 		rockset.WithCustomHeader("x-rockset-test", "go-client"),
 	}
 	var settings []govcr.Setting
-	path := vcrBucketPath(name)
-
-	// VCR cassettes are in S3 bucket rockset-go-tests (in us-west-2)
-	cfg, err := config.LoadDefaultConfig(context.Background(), config.WithRegion("us-west-2"))
-	if err != nil {
-		return nil, nil, err
-	}
-	s3Client := s3.NewFromConfig(cfg, func(o *s3.Options) { o.UsePathStyle = true })
+	path := fmt.Sprintf("./testing_assets/cassettes/%s.cassette.gz", name)
```

Comment:
I wonder if `testdata/cassettes` isn't a better place for them, as `testdata` is the idiomatic location for test data and it is ignored by all go tooling